<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportingWidget</class>
 <widget class="QWidget" name="ReportingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget #scrollAreaWidgetContents_2 { border : 3px ridge rgb(160, 160, 160) };</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="reportLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>... Report</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="reportPeriod">
     <property name="title">
      <string>... Period</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="startReportLabel">
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="startReportDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="displayFormat">
         <string>dd-MMM-yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="endReportLabel">
        <property name="text">
         <string>End:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="endReportDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="displayFormat">
         <string>dd-MMM-yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="generateReportButton">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="ReportScroll">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>416</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget #scrollAreaWidgetContents_2 { background-color: white};</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="reportBrowser">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>generateReportButton</tabstop>
  <tabstop>startReportDate</tabstop>
  <tabstop>endReportDate</tabstop>
  <tabstop>reportBrowser</tabstop>
  <tabstop>ReportScroll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
